.github/ISSUE_TEMPLATE/1_bug_report.yml

```
@@ -5,11 +5,12 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking the time to fill out this bug report. Please take the time to provide as much details as possible. Following our [bug report guidelines](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) will help maintainers qualify your report faster.
-
-        Reporting bugs with "error 500" or "white page" may not help us reproduce them. This is a universal error that hides what is underneath. Instead, enable debug mode in the Performance section of PrestaShop and try again. You should get a proper error message.
-
-        🇬🇧 Use English, reports in all other languages will be closed.
+         #### Basic information ❗
+         - Provide as much DETAILS as possible - error logs, screenshots, your exact configuration. If no one can reproduce it, no one will fix it.
+         - Follow [bug report guidelines](https://devdocs.prestashop-project.org/8/contribute/contribute-reporting-issues/#best-practices-for-writing-an-issue) to help maintainers qualify your report faster.
+         - DO NOT report bugs with "error 500" or "white page" - this is a universal error and will tell us nothing. Enable debug mode in the Performance section of PrestaShop or manually in `/config/defines.inc.php` and try again. You should get a proper error message.
```

It’s better to not talk like that 😊 « DO NOT… » may sound aggressive

---

I know Nicolas, but people still open bugs like this. :(

I think we should make an exception on our "polished speech" policy. We have nothing to prove to anybody, we need good bug report. `Reporting bugs with "error 500" or "white page" may not help us reproduce them.` may sound polite, but it's not what it means. It will tell us zero information.

- https://github.com/PrestaShop/PrestaShop/issues/33482
- https://github.com/PrestaShop/PrestaShop/issues/33306
- https://github.com/PrestaShop/PrestaShop/issues/33268

---

I understand but I don’t agree 😉

---

I'd go for `Please DO NOT  ... ` to make everyone happy 🇨🇭 😅

---

I think it's possible to use `**bold**` instead of SCREAMING
```suggestion
         - Avoid reporting "error 500" or "white page" errors - this is a universal error message that does not provide enough information to qualify the issue. Enable debug mode in the Performance section of PrestaShop or manually in `/config/defines.inc.php` and try again. You should get a proper error message.
```